docs/running-on-yarn.md

```diff
@@ -118,19 +118,6 @@ To use a custom metrics.properties for the application master and executors, upd
   </td>
 </tr>
 <tr>
-  <td><code>spark.driver.memory</code></td>
```

Comment:
Yes, I see that these are duplicated in the YARN docs and main config docs. So are things like `spark.driver.cores`, but, the info on what it means in YARN is slightly more specific to YARN in the YARN docs. The `spark.driver.memory` doc isn't different, but, for consistency, maybe does not hurt to leave a note about these key, generic props in the YARN docs.